julia-1.2.0-full.tar.gz does not build behind firewall

As This worked fine when I downloaded and built Julia 1.0 from source.  However, when I download the julia-1.2.0-full.tar.gz file from https://github.com/JuliaLang/julia/releases/download/v1.2.0/julia-1.2.0-full.tar.gz, I can't get it to build on an isolated network.  Currently the steps I follow are:

% tar -xvf julia-1.2.0-full.tar.gz
% make
Warning: git information unavailable; versioning information limited
Warning: git information unavailable; versioning information limited
Warning: git information unavailable; versioning information limited
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (7) Failed connect to cache.julialang.org:443; Connection refused
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (7) Failed connect to github.com:443; Connection refused
make[1]: *** [/home/user1/julia-1.2.0/deps/srccache/SuiteSparse.v5.4.0-2.x86_64-linux-gnu-gcc4.tar.gz] Error 7
make: *** [julia-deps] Error 2


OS:  RHEL 7.6 fully patched.


Any help would be greatly appreciated.

--- a/doc/build/build.md
+++ b/doc/build/build.md
@@ -17,8 +17,8 @@ variables.
 When compiled the first time, the build will automatically download
 pre-built [external
 dependencies](#required-build-tools-and-external-libraries). If you
-prefer to build all the dependencies on your own, add the following in
-`Make.user`
+prefer to build all the dependencies on your own, or are building on a system that cannot
+access the network during the build process add the following in `Make.user`:
 
 ```
 USE_BINARYBUILDER=0

--- a/doc/build/build.md
+++ b/doc/build/build.md
@@ -18,7 +18,7 @@ When compiled the first time, the build will automatically download
 pre-built [external
 dependencies](#required-build-tools-and-external-libraries). If you
 prefer to build all the dependencies on your own, or are building on a system that cannot
-access the network during the build process add the following in `Make.user`:
+access the network during the build process, add the following in `Make.user`:
 
 ```
 USE_BINARYBUILDER=0

